### DISCUSSION TITLE
AMD HEVC support?

### DISCUSSION THREAD

**Discussion by Andy76swe** [mifi/lossless-cut#1767] [Q&A]
Would it be difficult to add support for HEVC on AMD GPUs/CPUs? I used to use this program on an i9900k and it worked flawlessly.. swapped to an AMD 7800x3D and now it just shows a black screen when HEVC decoding is enabled. Same GPU in both cases, RX 6900XT.

**Comment by mifi** [comment]
Not sure which OS you are on, but this is nothing losslesscut can support, it all comes down to chromium/electron's support (as well as the operating system)

**Comment by thedustyrover** [comment]
Ah dang, just ran into this. Couple questions:

 * is there no way to fall back on software encoding for HEVC even if slow?
 * I guess it'd be helpful if lossless cut could mention this in the error message (AMD CPU/GPU does not have support for HVEC).

**Reply by mifi** [reply]
what's the error message youre getting?

**Reply by thedustyrover** [reply]
"Unable to playback this file. Try to convert to supported format from the menu."

Upon further digging, it is because these files are 10-bit HEVC (from an iPhone), which has no hardware support from AMD (I'm on an Ryzen 7 6800U). A normal 8-bit HEVC works okay. Not sure what that means for a software fallback being possible.

fwiw the files play fine on my Windows Media Player, VLC, etc. Perhaps a problem with Lossless-Cut not having access to the same installed codecs?

**Reply by mifi** [reply]
Ok, you can try to go to settings and disable "Hardware HEVC decoding", and you can try File-Convert to supported format "Fastest: FFmpeg-assisted playback" - then it will use slower software decoding using ffmpeg. LosslessCut uses Chromium for playback, so it's probably lacking support for your hardware H265 decoder (for 10bit) in Chromium. Probably related: #2228
